versions.yaml

```diff
@@ -297,14 +297,14 @@ externals:
   virtiofsd:
     description: "vhost-user virtio-fs device backend written in Rust"
     url: "https://gitlab.com/virtio-fs/virtiofsd"
-    version: "v1.3.0"
-    toolchain: "1.62.0"
+    version: "v1.6.1"
+    toolchain: "1.66.0"
     meta:
-      # From https://gitlab.com/virtio-fs/virtiofsd/-/releases/v1.3.0,
-      # this is the link labelled virtiofsd-v1.3.0.zip
+      # From https://gitlab.com/virtio-fs/virtiofsd/-/releases/v1.6.0,
+      # this is the link labelled virtiofsd-v1.6.0.zip
```

Comment:
Should these comment links be pointing at v1.6.1 rather than v1.6.0 if that's what we are using?

Reply:
Yep, it should! :-)

Reply:
As it was my fault updating the tarball to v1.6.1, but forgetting about the comments, I've updated it.